crates/node/src/migration_service/types.rs

```diff
@@ -0,0 +1,55 @@
+use ed25519_dalek::VerifyingKey;
+use mpc_contract::node_migrations::{BackupServiceInfo, DestinationNodeInfo};
+use near_sdk::AccountId;
+
+use crate::{indexer::migrations::ContractMigrationInfo, providers::PublicKeyConversion};
+
+#[derive(PartialEq, Debug, Clone)]
+pub struct MigrationInfo {
+    pub backup_service_info: Option<BackupServiceInfo>,
+    pub active_migration: bool,
+}
+
+impl MigrationInfo {
+    pub fn from_contract_state(
+        my_account_id: &AccountId,
+        my_p2p_tls_key: &VerifyingKey,
```

Comment:
If this type is not actually used for some crypto operation here, then I think it is better to use the dtos_contract type: `Ed25519PublicKey`

Reply:
Good observation!
This is the type currently exposed by the contract API. We have an issue to change it, but that is out of scope for this PR: https://github.com/near/mpc/issues/1173.

Reply:
Interesing :) I thought I had changed that, but maybe the change is not in main yet.

Reply:
Indeed it was changed https://github.com/near/mpc/pull/1225